File: library/src/main/java/org/rustlang/core/panicking.java
package org.rustlang.core;

public final class panicking {
    private panicking() {}

    public static void panic(String arg) {
        throw new RuntimeException("Rust panic: " + arg);
    }

    public static void panic_fmt(Object args) {
        throw new RuntimeException("Rust panic: " + args);
    }
}

File: library/src/main/kotlin/org/rustlang/core/panicking/panicking.kt
package org.rustlang.core 

public object panicking {
    @JvmStatic
    public fun panic(arg: String?) {
        throw RuntimeException("Rust panic: $arg")
    }

    @JvmStatic
    public fun panic_fmt(args: Any?) {
        throw RuntimeException("Rust panic: $args")
    }
}